README.md

```diff
@@ -44,6 +44,12 @@ More information about running `docker-compose` can be found [here.](https://doc
 
 [Note : Cloner clones the whole website, to restrict to a desired depth of cloning add `--max-depth` parameter]
 
+In case of an error while running `docker-compose up`, check the port 80 by running `sudo netstat -tulnp`, if they are occupied then change the 
```

Comment:
I am sorry but instructions for docker is out of the score for the project. As I said before, you can only mention, that Snare will start on 0.0.0.0 with port 80 and that's it. e.g. as a point 4 here https://github.com/mushorg/snare/pull/253/files#diff-04c6e90faac2675aa89e2176d2eec7d8R41